packages/manager/apps/ips/src/pages/byoip/ByoipOrder.component.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ByoipContext } from './Byoip.context';
+import { RirSelectionSection } from './sections/RirSelectionSection.component';
+import { RegionSelectionSection } from './sections/RegionSelectionSection.component';
+import { TokenSelectionSection } from './sections/TokenSection.component';
+import { IpRangeSelectionSection } from './sections/IpRangeSelectionSection.component';
+
+export const ByoipOrder: React.FC = () => {
+  const { ipRir, selectedRegion, selectedToken, ipRange } = React.useContext(
+    ByoipContext,
+  );
+
+  const visibleSections = {
+    ipRirSelection: ipRir,
+    regionSelection: selectedRegion,
```

Comment:
regionSelection appears when ipRir has been selected so you should put the ipRir condition here

Reply:
yeah I will modify it to have correct conditions